satpy/writers/cf_writer.py

```diff
@@ -694,6 +716,9 @@ def save_datasets(self, datasets, filename=None, groups=None, header_attrs=None,
         for kwarg in satpy_kwargs:
             to_netcdf_kwargs.pop(kwarg, None)
 
+        # Allow to prepend CHANNEL_ to datasets name staring with digit
+        valid_cf_dataset_name = to_netcdf_kwargs.pop('valid_cf_dataset_name', False)
```

Comment:
Why is this not a defined keyword argument in the method definition?

Reply:
I was not sure how I should do it. Adding it as a keyword would increase the complexity if the method definition, but better the readability.